test/functional/services/remote/browserdriver_api/browserdriver_api.js

```diff
@@ -1,21 +1,21 @@
 import { EventEmitter } from 'events';
 
-import { createLocalChromedriverApi } from './chromedriver_local_api';
-import { createRemoteChromedriverApi } from './chromedriver_remote_api';
+import { createLocalBrowserdriverApi } from './browserdriver_local_api';
+import { createRemoteBrowserdriverApi } from './browserdriver_remote_api';
 import { ping } from './ping';
 
 const noop = () => {};
 
 /**
- *  Api for interacting with a local or remote instance of Chromedriver
+ *  Api for interacting with a local or remote instance of a browser
  *
  *  @type {Object}
  */
-export class ChromedriverApi extends EventEmitter {
-  static async factory(log, url) {
+export class BrowserdriverApi extends EventEmitter {
+  static async factory(log, url, browserName) {
```

Comment:
Since we're using `TEST_BROWSER_TYPE`, and `browserType` in a couple other places, we should probably go with `browserType` here too. Also, it's renamed to `browser` again below.

Reply:
Done